js/src/views/Account/Header/header.js

```diff
@@ -67,6 +71,10 @@ export default class Header extends Component {
             <Balance
               account={ account }
               balance={ balance } />
+            <Certifications
+              certifications={ certifications }
+              dappsUrl={ api.dappsUrl }
```

Comment:
From what I can tell, `certifications` is a Redux state, thus we could connect `Certifications` to the state and make it _dumb_